docs/espace/tutorials/scaffoldCfx/scaffold.md

```diff
@@ -0,0 +1,131 @@
+
+```bash
+yarn deploy --network ConfluxEspace
+```
+
+```bash
+yarn deploy --network confluxEspaceTest
+```
+
+:::tip
+The deployer of the contract and thereby the owner of the contract can be changed by re-labelling the .env.example to .env and putting your private key in DEPLOYER_PRIVATE_KEY=. However, the default uses a default "public" private key for hardhat testing purposes.
+:::
+
+Once you have deployed the contract, you can start up your front-end to see how it interacts with the contracts. You should be able to see an interface like the following. The Contract Debug page is similar to other web deployment applications like Remix.
+
+![Front-end](../img/FrontEnd.png)
+
+What is powerful about Conflux Scaffold is that you can quickly deploy your contract in Hardhat and be able to see both the transactions on a block explorer and interact with the read/write functions of the contract
+
+![Contract](../img/ContractInteraction.png)
+
+
+## Front-End
+
+Once you have your front-end up and running, the fun begins on connecting your front-end with the contract. You can leverage beautiful react components.
+
+- Navigate to conflux-scaffold/packages/nextjs/app/page.tsx
+
+![Import Libraries](../img/Components.png)
+
+- You can see here that components can be quickly imported from /components/scaffold-eth
+- For example, there is an Address, Balance, AddressInput component, etc. all which can be used quickly to add to your app. In the video tutorial, there is an example of how an AddressInput is quickly imported and rendered.
+- Additional components can be seen in the documentation https://docs.scaffoldeth.io/components/
+- There are other hooks which are extremely important for reading and writing the contract (as well as setting up event subscription). These are the useScaffoldContractRead, useScaffoldContractWrite, etc. which are imported from "~~/hooks/scaffold-eth/index"
+- The arguments requirement for the hooks are all in the [documentation](https://docs.scaffoldeth.io/components/) but here are two examples of how easy it is to create a contract instance, read a function from the contract, and write into a function.
+
+![Contract Instance](../img/ContractInstance.png)
+
+![Contract Read](../img/ContractRead.png)
+
+![Contract Write](../img/ContractWrite.png)
+
+## Deployment of Conflux Contracts
+
+In order to deploy a conflux app, we need to first deploy the contract into Conflux Espace and allow our wallet to connect to Conflux ESpace. In conflux-scaffold/packages/hardhat/.env.example, we need to change this to packages/hardhat/.env and include our private key in DEPLOYER_PRIVATE_KEY. This makes our private key as the deployer of the contract. In the solidity contract, we have also assigned the deployer as the owner of the contract.
+
+```bash
+yarn deploy --network ConfluxEspace
+```
+
+or 
+
+```bash
+yarn deploy --network ConfluxEspaceTest
```

Comment:
This network configuration is defined in hardhat. I suggest the network definition would be better to be consistent with viem definition

```suggestion
yarn deploy --network confluxESpaceTest
```